spec/setup_test_db.rb

```diff
@@ -3,11 +3,37 @@
 def setup_test_database
   connection = PG.connect(dbname: 'ducks_bnb_test')
   connection.exec('TRUNCATE spaces CASCADE;')
+  connection.exec('TRUNCATE users CASCADE;')
+  connection.exec('TRUNCATE bookings CASCADE;')
```

Comment:
I think here, because we are truncating all tables, we don't need the cascade method on each.

Reply:
I have changed this to drop table now and left the cascade in as it was throwing an error if not.